lib/adal.js

```diff
@@ -482,9 +477,15 @@ var AuthenticationContext = (function () {
     };
 
     /**
+     * @callback tokenCallback
+     * @param {string} error error message returned from AAD if silent token request fails.
+     * @param {string} token token returned from AAD if silent token request is successful.
```

Comment:
We should remove the 'silent' word, as the same callback will be mostly be used for interactive flows too, whenever we add them.